activities/FractionBounce.activity/js/components/SugarDevice.js

```diff
@@ -33,21 +32,92 @@ Vue.component('sugar-device', {
 		}
 	},
 	methods: {
+		isMobile() {
+			var userAgent = navigator.userAgent;
+			if (/Mobile|mini|Fennec|Android|iP(ad|od|hone)/.test(userAgent)) {
+				return true;
+			}
+			return false;
+		},
+		isElectron() {
+			var userAgent = navigator.userAgent;
+			if (/Electron/.test(userAgent)) {
+				return true;
+			}
+			return false;
+		},
+		isWebApp() {
+			if (!this.isMobile() && !this.isElectron()) {
```

Comment:
It's not true, if it's launched using `file:///` it's not mobile, it's not electron but it's not a WebApp neither.
I think you should consider adding a `ifApp` method using the same condition than [here](https://github.com/llaske/sugarizer/blob/dev/lib/util.js#L237).

Reply:
Added the simple `isApp()` method.